src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -75,6 +78,10 @@ private ResponseEntity postEventInternal(String eventTypeName, String eventsAsSt
         }
     }
 
+    private ThrowableProblem crateProblem(JSONException e) {
+        return Problem.valueOf(Response.Status.BAD_REQUEST, e.getMessage());
```

Comment:
You can look at it: https://github.com/stleary/JSON-java
Also it is not clear how to implement it according to ARUHA-162 because if we show 'Your json request is not array' for every JSONException it will be wrong. We can check for the array our selfs but I think it is more cumbersome